src/ui/wxWidgets/ExternalKeyboardButton.cpp

```diff
@@ -68,15 +68,15 @@ void ExternalKeyboardButton::HandleCommandEvent(wxCommandEvent& evt)
   wxString command = wxString(wxT("xvkbd"));
 
   if (!pws_os::ProgramExists(tostdstring(command))) {
-    wxMessageBox(_("Could not launch xvkbd.  Please make sure it's installed and in your PATH"), 
+    wxMessageBox(_("Could not launch xvkbd.  Make sure it's installed and in your PATH"), 
```

Comment:
Same about the three lines changed in this file

Reply:
agree.
"Please make sure it's ..."